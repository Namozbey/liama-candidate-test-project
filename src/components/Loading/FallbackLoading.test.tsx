import React from "react";
import { render } from "@testing-library/react";
import { FallbackLoading } from ".";

describe("Testing FallbackLoading", () => {
  it("Snapshots", () => {
    const { asFragment } = render(<FallbackLoading />);
    expect(asFragment()).toMatchSnapshot();
  });
});
