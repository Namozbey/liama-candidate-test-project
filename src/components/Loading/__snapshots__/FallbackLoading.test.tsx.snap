// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing FallbackLoading Snapshots 1`] = `
<DocumentFragment>
  <div
    class="fallback-loading"
  >
    Loading...
  </div>
</DocumentFragment>
`;
